### ISSUE TITLE
Ext components within arbitrary/ignored blocks within other Ext components breaks rendering

### ISSUE THREAD

**Issue by fabriciomurta** [extnet/Ext.NET#1757]
The whole web page won't render without any error message (just blank response) so becoming hard to troubleshoot the issue.

Reproduced at: [editor/basic/overview/index.cshtml#L143-L162](https://github.com/objectdotnet/ext7-sandbox/blob/1e6994ea47e7aed4bd7e57b0e5e9a132dd5538b9/ExtNetCore7-20200325/Pages/examples/editor/basic/overview/index.cshtml#L143-L162)

A minimialistic code to reproduce this issue is as follows:
```xml
<ext-container width="200" height="200">
    <content>
        Test
        <ext-component html="Test" />
    </content>
</ext-container>
```

Arbitrary blocks applies:

```xml
<ext-container>
    <somethingunsupported>
        somethingrandom
        <ext-component html="random component" />
    </somethingunsupported>
</ext-container>
```

Any page containing code like this won't render at all, receiving an empty response from the server instead of any error response.

**Comment by AndreyChechel** [comment]
The first sample with the inner `<content>` tag was fixed within #1755
The empty response issue was fixed within #1758

The **arbitrary blocks** sample now produces the following exception:
```
LayoutException: Unexpected element 'ComponentTagHelper' nested into 'ContainerTagHelper'.

Ext.Net.Core.TagHelpers.ExtTagHelper.SetChildModel(IExtModelHelper child) in 
    ExtTagHelper.cs, line 45
    ...
```

@fabriciomurta it seems the exception is valid, since a container's inner components must be included into inner `<items></items>` block. Do you think it's fine, or is there any expected output you'd like to see?

**Comment by fabriciomurta** [comment]
yes, throwing an exception is much better than showing nothing at all in the whole page; and also better than just ignoring silently the extra inside contents/tags/blocks. Feel free to close this if the behavior you described is already actual in Ext.NET 7
